docs/source/generic/rest.rst

```diff
@@ -42,9 +42,8 @@ altijd een JSON response geforceerd.
 Paginering
 ----------
 
-De resultaten hebben een standaard paginagrootte van 1000 records.
-
-De paginagrootte kan aangepast worden door een :samp:`&page_size={n}` query parameter toe te voegen aan de request URL.
+De resultaten worden gepagineerd teruggegeven.
+De paginagrootte kan aangepast worden door een :samp:`?page_size={n}` query parameter toe te voegen aan de request URL.
```

Comment:
Oops.. deze parameter mag zelfs omgezet worden naar `?_pageSize={n}`. De `page_size` is de oude parameter die we voor backwords compatibiliteit nog laten werken.

Reply:
Done